superset-frontend/package.json

```diff
@@ -140,6 +140,7 @@
     "re-resizable": "^4.3.1",
     "react": "^16.13.0",
     "react-ace": "^5.10.0",
+    "react-avatar": "^3.9.7",
```

Comment:
Is this something that we might want to migrate away from in the future if the Antd discussions we were talking about in the meetup today come to fruition? I see that antd has an avatar component already: https://ant.design/components/avatar/

Reply:
Possibly, but we can't block forward progress based on a pre-SIP discussion for something that may or may not pass a vote. Should AntD come to fruition, we can swap out components later.

Reply:
I'm not saying to block progress, moreso wondering if since we're adding a new library anyway if we should think ahead. Either way, whichever you think is best works